manifest: Updated Matter repository to pull commissioning fix (#17345)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-7-0

> KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, what violates the Matter specification.

Reviewer: Can we add information that the issue impacts following Matter Certification Test Cases:
[TC-CADMIN-1.21] Open commissioning window - durations max and max+1 [BCM] [DUT - Commissionee]
[TC-CADMIN-1.22] Open commissioning window - durations max and max+1 [ECM] [DUT - Commissionee]

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-7-0

> KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, what violates the Matter specification.

Reviewer 2: ```suggestion
KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, which violates the Matter specification
```

>   The device using Extended Announcement feature is allowed to open commissioning window for longer than 900 s only if it's uncommissioned.

Reviewer 2: ```suggestion
  A device using the Extended Announcement feature is allowed to open the commissioning window for longer than 900 s only if it is uncommissioned.
```

>   However, the implementation with Extended Announcement enabled allows to open commissioning window for longer than 900 s if the device is commissioned as well.

Reviewer 2: ```suggestion
  However, the implementation with Extended Announcement enabled allows the device to open the commissioning window for longer than 900 s even when the device is commissioned.
```

>   It leads to the TC-CADMIN-1.21 and TC-CADMIN-1.22 certification test cases failure.

Reviewer 2: ```suggestion
  This leads to the TC-CADMIN-1.21 and TC-CADMIN-1.22 certification test cases failure.
```
